packages/ra-example/posts.js

```diff
@@ -82,6 +82,14 @@ export const PostList = ({ ...props }) => (
         {...props}
         filters={<PostFilter />}
         sort={{ field: 'published_at', order: 'DESC' }}
+        selectable
+        selectMode="bulk"
+        selectActions={[
```

Comment:
Not fan of the actions as objects here. It would be better to pass a component like we do for `actions` already.
Maybe we can somehow just use the current `actions` but passing it a selection of items as props ?

Reply:
So I thought about this a bit. Using a selectActions attribute allows the usage of actions without implementing a custom Actions component. It's just for ease of use. The SelectActionButton requires to know what actions to render. Maybe this can be defaulted to a specific list (delete) and if the user wants to overwrite a custom actions component can be used.

Reply:
I think we can handle that without a new `selectActions` prop just by passing the currently selected items to the existing component specified by the `actions` prop [here](https://github.com/marmelab/admin-on-rest/blob/master/src/mui/list/List.js#L246). We might provide a default `Delete` button handling this case in the default [`Actions` component](https://github.com/marmelab/admin-on-rest/blob/master/src/mui/list/Actions.js).

Reply:
I think I know what you mean. I'll refactor a bit and come back